README.md

```diff
@@ -83,6 +83,16 @@ Deploy the site by running:
 3. Visit [`0.0.0.0:8000/stylguide`](http://0.0.0.0:8000/styleguide) for the style guide
 4. Visit [`0.0.0.0:8000/a11y`](http://0.0.0.0:8000/a11y) for an accessibility report
 
+#### Tasks
+
+- `npm install` - Install local dependencies
+- `npm run build` - Builds the site and the styleguide
+- `npm test` - Builds the site, styleguide, and runs reports (a11y)
+- `npm start` - Watches and serves the site for local development
+- `npm run deploy` - Deploys the site to what is defined in the "deploy" configuration
```

Comment:
*Warning*: Running `npm run deploy` will deploy the site. Don't do this if you don't want to deploy the site.

If you have some better copy for this, then do it.